### PR TITLE
CCM: use flagbuilder instead of manually building argv

### PR DESCRIPTION
**Pull request by jiahuif**
Because CCM did not use flagbuilder, flags defined in the configuration were not applied. Rewrite the argv building to use standard approach.  Also reorder flags in tests, and add tests for other flags.

/kind feature

### CODE REVIEW

**Comment by jiahuif** [comment]
Oops forgot bazel. Let me try again.

File: upup/pkg/fi/cloudup/template_functions_test.go [hunk 1]
```diff
@@ -77,8 +77,8 @@ func Test_TemplateFunctions_CloudControllerConfigArgv(t *testing.T) {
 				},
 			},
 			expectedArgv: []string{
-				"--v=3",
 				"--cloud-provider=openstack",
+				"--v=3",
```
Comment (jiahuif) [inline upup/pkg/fi/cloudup/template_functions_test.go:81]: Only the ordering was changed. It would be easier to sort + unique.

File: tests/integration/update_cluster/many-addons-ccm-irsa/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content [hunk 2]
```diff
@@ -20,12 +20,13 @@ spec:
...
       - args:
-        - --v=2
+        - --allocate-node-cidrs=true
         - --cloud-provider=aws
-        - --cluster-name=minimal.example.com
         - --cluster-cidr=172.20.0.0/16
-        - --allocate-node-cidrs=true
+        - --cluster-name=minimal.example.com
         - --configure-cloud-routes=false
+        - --leader-elect=true
```
Comment (jiahuif) [inline tests/integration/update_cluster/many-addons-ccm-irsa/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content:28]: Kind of scary to think of previous HA clusters not having leader election on ... oh well.

Comment (jiahuif) [reply tests/integration/update_cluster/many-addons-ccm-irsa/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content:28]: Otherwise, only the ordering of the flags was changed.

**Comment by jiahuif** [comment]
assign/cc authors of related PRs
/assign @johngmyers @rifelpet @olemarkus
/cc @DavidSie @justinsb

File: tests/integration/update_cluster/minimal-1.24/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content [hunk 3]
```diff
@@ -20,12 +20,14 @@ spec:
...
       - args:
-        - --v=2
-        - --cloud-provider=aws
-        - --cluster-name=minimal.example.com
-        - --cluster-cidr=100.64.0.0/10
         - --allocate-node-cidrs=true
+        - --cluster-cidr=100.64.0.0/10
+        - --cluster-name=minimal.example.com
         - --configure-cloud-routes=false
+        - --enable-leader-migration=true
```
Comment (jiahuif) [inline tests/integration/update_cluster/minimal-1.24/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content:27]: This is intended - Leader Migration is enabled by default to CCM, as of 1.24

**Comment by jiahuif** [comment]
With this change, I have successfully gone through a full Leader Migration on GCP.
Also cc'ing previous reviewers or users.
cc @zetaab @rifelpet

**Comment by jiahuif** [comment]
source.yaml
```yaml
apiVersion: kops.k8s.io/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: null
  name: hacluster.k8s.local
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudConfig: {}
  cloudProvider: gce
  configBase: gs://jhf-kubernetes-clusters/hacluster.k8s.local
  #  cloudControllerManager:
  #  enableLeaderMigration: true
  #  image: k8scloudprovidergcp/cloud-controller-manager:v1.23.3
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: main
  - cpuRequest: 100m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: events
  iam:
    allowContainerRegistry: true
    legacy: false
  kubelet:
    anonymousAuth: false
  kubernetesApiAccess:
  - 0.0.0.0/0
  - ::/0
  kubernetesVersion: 1.23.1
  masterPublicName: api.hacluster.k8s.local
  networking:
    kubenet: {}
  nonMasqueradeCIDR: 100.64.0.0/10
  project: jhf-gke-dev
  sshAccess:
  - 0.0.0.0/0
  - ::/0
  subnets:
  - cidr: 10.0.16.0/20
    name: us-central1
    region: us-central1
    type: Public
  topology:
    dns:
      type: Public
    masters: public
    nodes: public

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-a
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-b
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-c
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-c

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-a
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-b
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-c
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-c

```
dest.yaml
```yaml
apiVersion: kops.k8s.io/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: null
  name: hacluster.k8s.local
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudConfig: {}
  cloudProvider: gce
  configBase: gs://jhf-kubernetes-clusters/hacluster.k8s.local
  kubeControllerManager:
     enableLeaderMigration: false # no Leader Migration for the new KCM
  cloudControllerManager:
     enableLeaderMigration: true
     image: k8scloudprovidergcp/cloud-controller-manager:v1.23.3
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: main
  - cpuRequest: 100m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: events
  iam:
    allowContainerRegistry: true
    legacy: false
  kubelet:
    anonymousAuth: false
  kubernetesApiAccess:
  - 0.0.0.0/0
  - ::/0
  kubernetesVersion: 1.23.3
  masterPublicName: api.hacluster.k8s.local
  networking:
    kubenet: {}
  nonMasqueradeCIDR: 100.64.0.0/10
  project: jhf-gke-dev
  sshAccess:
  - 0.0.0.0/0
  - ::/0
  subnets:
  - cidr: 10.0.16.0/20
    name: us-central1
    region: us-central1
    type: Public
  topology:
    dns:
      type: Public
    masters: public
    nodes: public

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-a
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-b
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-c
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-c

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-a
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-b
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-c
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-c

```

**Comment by jiahuif** [comment]
Here are the cluster defs I used for the migration.
source.yaml
```yaml
apiVersion: kops.k8s.io/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: null
  name: hacluster.k8s.local
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudConfig: {}
  cloudProvider: gce
  configBase: gs://CHANGEME/hacluster.k8s.local
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: main
  - cpuRequest: 100m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: events
  iam:
    allowContainerRegistry: true
    legacy: false
  kubelet:
    anonymousAuth: false
  kubernetesApiAccess:
  - 0.0.0.0/0
  - ::/0
  kubernetesVersion: 1.23.1
  masterPublicName: api.hacluster.k8s.local
  networking:
    kubenet: {}
  nonMasqueradeCIDR: 100.64.0.0/10
  project: CHANGEME
  sshAccess:
  - 0.0.0.0/0
  - ::/0
  subnets:
  - cidr: 10.0.16.0/20
    name: us-central1
    region: us-central1
    type: Public
  topology:
    dns:
      type: Public
    masters: public
    nodes: public

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-a
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-b
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-c
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-c

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-a
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-b
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-c
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-c

```
dest.yaml
```yaml
apiVersion: kops.k8s.io/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: null
  name: hacluster.k8s.local
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudConfig: {}
  cloudProvider: gce
  configBase: gs://CHANGEME/hacluster.k8s.local
  kubeControllerManager:
     enableLeaderMigration: false # no Leader Migration for the new KCM
  cloudControllerManager:
     enableLeaderMigration: true
     image: k8scloudprovidergcp/cloud-controller-manager:v1.23.3
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: main
  - cpuRequest: 100m
    etcdMembers:
    - instanceGroup: master-us-central1-a
      name: a
    - instanceGroup: master-us-central1-b
      name: b
    - instanceGroup: master-us-central1-c
      name: c
    memoryRequest: 100Mi
    name: events
  iam:
    allowContainerRegistry: true
    legacy: false
  kubelet:
    anonymousAuth: false
  kubernetesApiAccess:
  - 0.0.0.0/0
  - ::/0
  kubernetesVersion: 1.23.3
  masterPublicName: api.hacluster.k8s.local
  networking:
    kubenet: {}
  nonMasqueradeCIDR: 100.64.0.0/10
  project: CHANGEME
  sshAccess:
  - 0.0.0.0/0
  - ::/0
  subnets:
  - cidr: 10.0.16.0/20
    name: us-central1
    region: us-central1
    type: Public
  topology:
    dns:
      type: Public
    masters: public
    nodes: public

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-a
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-b
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: master-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: master-us-central1-c
  role: Master
  subnets:
  - us-central1
  zones:
  - us-central1-c

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-a
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-a
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-a

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-b
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-b
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-b

---

apiVersion: kops.k8s.io/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: null
  labels:
    kops.k8s.io/cluster: hacluster.k8s.local
  name: nodes-us-central1-c
spec:
  image: ubuntu-os-cloud/ubuntu-2004-focal-v20211118
  machineType: e2-medium
  manager: CloudGroup
  maxSize: 1
  minSize: 1
  nodeLabels:
    cloud.google.com/metadata-proxy-ready: "true"
    kops.k8s.io/instancegroup: nodes-us-central1-c
  role: Node
  subnets:
  - us-central1
  zones:
  - us-central1-c

```

**Review by zetaab — APPROVED** [review]
